requirements/requirements.txt

```diff
@@ -61,3 +61,4 @@ tempita==0.5.2            # via sqlalchemy-migrate
 uritemplate==3.0.1        # via coreapi
 urllib3==1.25.7           # via requests
 vine==1.3.0               # via amqp, celery
+django-reversion==3.0.8
```

Comment:
this should be added to `requirements.in` (and then you should use the command at the top of this file to re-generate it and commit that)